<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>559</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*/344854/*/

QWidget{
	background-color: #FFFFFF;
}
#Form{
	background-color: rgb(255, 255, 255);
}

QLineEdit{
	background-color:#EFF3F6;
	
	color: rgb(28, 40, 47);
	border-radius: 7px;

}
 QDialogButtonBox &gt; QPushButton[text=&quot;&amp;OK&quot;] {
        background-color: orange;
    }
/*/---------------------------QPushButton------------------------&gt;&gt;/*/

QPushButton{
	
	background-color: rgb(28, 40, 47);
	color: rgb(241, 242, 242);
	border-radius: 7px;
}


QToolButton{
	
	background-color: rgba(0, 0, 0, 0);


}


/*/---------------------------QScrollBar------------------------&gt;&gt;/*/

QScrollBar::vertical {
	background-color: rgb(220, 220, 220);
	margin:0px 0 0px 0;
}

QScrollBar::handle:vertical {
	background-color:rgb(28, 40, 47);
	border-radius: 5px;
}


QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical{
	background-color: rgb(239, 243, 246);
	margin:0px 0 0px 0;
}

QScrollBar::horizontal {
	background-color: rgb(220, 220, 220);
	margin:0px 0 0px 0;
}

QScrollBar::handle:horizontal {
	background-color:  rgb(220, 220, 220);
}

QScrollBar::handle:horizontal:hover{
	background-color:  rgb(220, 220, 220);
}

QScrollBar::handle:horizontal:pressed{
	background-color:  rgb(220, 220, 220);
}

QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal{
	background-color:  rgb(220, 220, 220);
}

/*/---------------------------QComboBox------------------------&gt;&gt;/*/

QComboBox{
	color: rgb(28, 40, 47);
	background-color:rgb(239, 243, 246);
	padding-left: 10px;
	border-radius: 7px;
}

QComboBox::drop-down{
	border: 0px;
}

QComboBox::down-arrow{
	image: url(Data/PNG/caret-circle-down_1.png);
	width: 15px;
	height: 15px;
	margin-right: 15px
}

QComboBox QListView{
	border: 1px;
	color: rgb(28, 40, 47);
	background-color: rgb(239, 243, 246);
	outline: 0px
}

QComboBox QListView::item{
	padding-left: 10px;
	background-color: rgb(255, 0, 255);
}

QComboBox QListView::item::hover{
	background-color: rgb(255, 0, 0);
}

QComboBox QListView::item:selected{
	background-color: rgb(255, 0, 0);
}</string>
  </property>
  <widget class="QLineEdit" name="SelectPy">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiBold</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>Select file (.py)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SelectIco">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiBold</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="placeholderText">
    <string>Select icon (png, jpg, ico)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Ico">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>180</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiBold</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-top-left-radius: 0px;
border-bottom-left-radius: 0px</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Py">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>130</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiBold</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-top-left-radius: 0px;
border-bottom-left-radius: 0px</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Finsh">
   <property name="geometry">
    <rect>
     <x>176</x>
     <y>370</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiBold</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(80, 216, 144);

</string>
   </property>
   <property name="text">
    <string>Finsh</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_OneFile">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>270</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiBold</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>One file</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_Files">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>270</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiBold</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Files</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="Terminal">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>611</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiBold</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(239, 243, 246);
background-color: rgb(28, 40, 47);
border-radius: 7px;</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustIgnored</enum>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Bahnschrift SemiBold'; font-size:10pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Vilda TransPy Copyright (C) Vilda Corporation. All rights reserved. &lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Using Library 'pyinstaller'.&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Telegram: &lt;a href=&quot;t.me/Vilda_TM&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#50d890;&quot;&gt;t.me/Vilda_TM&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Github: &lt;a href=&quot;t.me/Vilda_TM&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#50d890;&quot;&gt;github.com/vildacode&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;============================&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="acceptRichText">
    <bool>true</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="SaveFile">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>230</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiBold</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="placeholderText">
    <string>Folder save (.exe)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_save">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>230</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiBold</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-top-left-radius: 0px;
border-bottom-left-radius: 0px</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Enter_name">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiBold</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="toolTipDuration">
    <number>1</number>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="placeholderText">
    <string>Name file .exe</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Copy">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>420</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiBold</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(28, 40, 47);
background-color: rgba(0, 0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Data/PNG/duplicate.png</normaloff>Data/PNG/duplicate.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Administrator">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>270</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiBold</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(28, 40, 47);
background-color: rgba(0, 0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Data/PNG/low2.png</normaloff>
     <normalon>Data/PNG/low1.png</normalon>
     <disabledon>Data/PNG/low1.png</disabledon>Data/PNG/low2.png</iconset>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>464</x>
     <y>270</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiBold</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Administrator</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>270</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiBold</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Remove console window</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Remove_Console">
   <property name="geometry">
    <rect>
     <x>276</x>
     <y>270</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiBold</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(28, 40, 47);
background-color: rgba(0, 0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Data/PNG/low2.png</normaloff>
     <normalon>Data/PNG/low1.png</normalon>
     <disabledon>Data/PNG/low1.png</disabledon>Data/PNG/low2.png</iconset>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_24">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiBold</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(28, 40, 47);
background-color: rgba(0, 0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Data/PNG/Untitled-1.png</normaloff>Data/PNG/Untitled-1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiBold</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiBold</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>File python:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiBold</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Icon:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiBold</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Folder save:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Button_Copy</sender>
   <signal>released()</signal>
   <receiver>Terminal</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>602</x>
     <y>433</y>
    </hint>
    <hint type="destinationlabel">
     <x>557</x>
     <y>418</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Button_Copy</sender>
   <signal>released()</signal>
   <receiver>Terminal</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>600</x>
     <y>422</y>
    </hint>
    <hint type="destinationlabel">
     <x>537</x>
     <y>418</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
